web/src/styles/sections/nav-bar.scss

```diff
@@ -28,6 +28,18 @@ $icon-offset: 45px;
   z-index: 5;
 }
 
+.nav-menu-item-container {
+  min-height: 47px;
+}
+
+.profile-default-background-color {
+  background-color: #fbfdf9;
+}
+
+.Mui-selected.bg-primary-extra-light-not-important {
+  background-color: #f2f7eb;
+}
```

Comment:
We were having issues make the normal bg-primary-extra-light class work because it has a !important on it which disables the MUI hover states. So I opted to do this instead of re-implementing the MUI hover states.

Reply:
I don't like this class name, but I see the reason why it exists. Even as a temporary implementation, I would prefer name it something else. Even just `background-primary-extra-light`.

Reply:
Moved [this ticket](https://www.pivotaltracker.com/story/show/185089904) to the top of the refactor backlog.